content/en/docs/compatibility-matrix.md

```diff
@@ -0,0 +1,82 @@
+| LDAP/Active Directory | All supported versions      |                                                                                 |
+| GitHub Teams          | All supported versions      | Roles from GitHub are mapped to the Teams under a specific GitHub organization. |
+| Google Groups         |                   |                                                                                 |
+## CI systems
+
+The following table lists the CI systems that Spinnaker supports:
+
+| Provider           | Spinnaker version      | Note |
+|--------------------|------------------------|------|
+| AWS CodeBuild      | 2.19.x or later        |      |
+| Google Cloud Build |                        |      |
+| Jenkins            | All supported versions |      |
+| Travis             | All supported versions |      |
+| Wercker            |                        |      |
+
+## Databases
+
+Depending on the service, Spinnaker can use either Redis or MySQL as the backing store. The following table lists the supported database and the Spinnaker service:
+
+| Database 	| DB version 	| Spinnaker version 	| Spinnaker services 	|
+|----------	|------------	|-------------------	|--------------------	|
+| Redis    	|            	| All supported versions      	|                    	|
+| MySQL    	|            	| All supported versions      	|                    	|
+
+## Deployment targets
+
+| Vendor        | Deployment target               | Spinnaker version |
+|---------------|---------------------------------|-------------------|
+| Amazon AWS    | EC2, ECS, EKS                   | All supported versions      |
+| Cloud Foundry | PKS                             | All supported versions      |
+| Google Cloud  | App Engine, Compute Engine, GKE | All supported versions      |
+| Kubernetes    | Manifest-based deployments      | All supported versions      |
+
+## Metrics
+
+The following table lists the monitoring and analytics:
+
+| Provider   	| Spinnaker version 	| Note                                 	|
+|------------	|-------------------	|--------------------------------------	|
+| Prometheus 	| All supported versions      	| Paired with Grafana for dashboarding 	|
+| Datadog    	| All supported versions      	|                                      	|
+
+{{% alert title="Note" %}}Armory Enterprise for Spinnaker versions 2.20 and later require the Observability plugin for monitoring and analytics. Versions earlier than 2.20 use the Spinnaker monitoring daemon, which is deprecated. {{% /alert %}}
+
+## Secret stores
+
+The following table lists the secret stores that Spinnaker supports for referencing secrets in config files securely:
+
+| Provider             | Spinnaker version | Notes                                |
+|----------------------|-------------------|--------------------------------------|
+| AWS Secrets Manager  |                   |                                      |
```

Comment:
Before we add this one can we make sure there's docs for it?

Reply:
We've got some really light documentation on it, but it's there: https://docs.armory.io/docs/spinnaker-install-admin-guides/secrets/secrets-aws-sm/.

By request of @away168, I'm also going through and adding links to docs where it makes sense.